docs/products/flink/howto/create-jar-application.md

```diff
@@ -3,28 +3,19 @@ title: Create a JAR application
 limited: true
 ---
 
-Aiven for Apache Flink® enables you to upload and deploy
-[custom code as a JAR file](/docs/products/flink/concepts/custom-jars), enhancing your
-Flink applications with advanced data
-processing capabilities.
+Aiven for Apache Flink® enables you to upload and deploy [custom code as a JAR file](/docs/products/flink/concepts/custom-jars), enhancing your Apache Flink applications with advanced data processing capabilities.
 
 ## Prerequisite
 
 - Custom JARs for Aiven for Apache Flink is a **limited availability** feature.
-  To try this feature, request it by contacting
-  our [Sales team](mailto:sales@aiven.io).
-- Once activated on your Aiven account, enable it:
-  - For a new Aiven for Apache Flink service,
-    toggle the feature during service creation.
-  - For an existing service, in
-    [Aiven Console](https://console.aiven.io/):
-    1. Select your project and choose your Aiven for Apache Flink® service.
-    1. Click **Service settings** on the left sidebar.
-    1. Scroll to the **Advanced configuration** section, and click
-        **Configure**.
-    1. In the **Advanced configuration** screen, click **Add
-       configuration options**, and using the search box, find and set
-       `custom_code` configuration to **Enabled**.
+  To try this feature, request access by contacting the [sales team](mailto:sales@aiven.io).
+- Once activated on your Aiven account, create a new Aiven for Apache Flink service and
+toggle the option for **Upload and deploy custom JARs** to enable custom JARs during
+service creation.
```

Comment:
indention issue :) 

We can do a bit shorter with `click **Upload and deploy custom JARs** to enable` or `toggle **Upload and deploy custom JARs**`, no need to say it is an option, I'd argue?